README.md

```diff
@@ -13,5 +13,17 @@ mkdir /home/cheriot-rtos.mapped
 sudo bindfs --map=martha/boris /home/cheriot-rtos /home/cheriot-rtos.mapped
 ```
 
-VScode can now be launched with a dev container from /home/cheriot-rtos.mapped 
+VScode can now be launched with a dev container from /home/cheriot-rtos.mapped
+
+## Building locally
+
+One way to test the dev container is to use podman.
```

Comment:
Both Docker and Podman have the same CLI, so it would be nice to make these generic (something like:

```suggestion
You can test locally using either Docker or Podman.
The command-line interfaces for the two are the same, so just replace `podman` with `docker` in the examples if you are using Docker.
```

Reply:
Thanks, added this now.